Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -670,9 +728,14 @@ TextStream& operator<<(TextStream& ts, const InteractionRegion& interactionRegio
         ts.dumpProperty("content hint"_s, "photo"_s);
     auto radius = interactionRegion.cornerRadius;
     if (radius > 0) {
-        if (interactionRegion.maskedCorners.isEmpty())
+        if (interactionRegion.maskedCorners.isEmpty()) {
             ts.dumpProperty("cornerRadius"_s, radius);
-        else  {
+#if ENABLE(FORM_CONTROL_REFRESH)
+            auto useContinuousCorners = interactionRegion.useContinuousCorners;
+            if (useContinuousCorners)
+                ts.dumpProperty("useContinuousCorners"_s, useContinuousCorners);
```

Comment:
maybe? Since you're only ever logging the true case


```suggestion
            if (interactionRegion.useContinuousCorners)
                ts.dumpProperty("useContinuousCorners"_s, true);
```